platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,82 @@
+Platform Design Proposal Process
```

Comment:
Nit:

For this title, you use the `=======` markdown syntax to indicate `Heading Level 1`. But for the subsections of this document, you use a single `#`, which *also* indicates `Heading Level 1`. Therefore, your subsections have the same heading level as this document title

Recommend either using the `#` syntax for the title, and `##` for the subsections, or `=======` for the title, and `---------` for the subsections

Reply:
I will fix